projects/services-map-state/src/lib/map-state.service.ts

```diff
@@ -59,6 +59,28 @@ export class MapStateService {
     this.setMapState(state);
   }
 
+  public setViewAngle(angle: number, notifier: IMapState['options']['notifier'] = 'user') {
+    if (!angle) {
+      return;
+    }
+    this.lastAction.next('setAngle');
+    const state = this.getMapState().getValue();
+    state.options.notifier = notifier;
+    const newState = new MapState(state.zoom, state.center, state.options, state.extent, state.time, angle, state.rotation);
+    this.mapState.next(newState);
+  }
+
+  public setRotation(rotation: number, notifier: IMapState['options']['notifier'] = 'user') {
+    if (!rotation) {
```

Comment:
In here, I would allow to pass 0 as rotation. Otherwise, users must use small values.